src/options.cpp

```diff
@@ -39,8 +39,8 @@ std::map<int, std::string> Option::s_systemDivider = { { SYSTEMDIVIDER_none, "no
 constexpr const char *engravingDefaults
     = "{'engravingDefaults':{'thinBarlineThickness':0.15,'lyricLineThickness':0.125,"
       "'slurMidpointThickness':0.3,'staffLineThickness':0.075,'stemThickness':0.1,'tieMidpointThickness':0.25,"
-      "'hairpinThickness':0.1,'thickBarlineThickness':0.5,'tupletBracketThickness':0.1,'subBracketThickness':0.5,"
-      "'bracketThickness':0.5,'repeatEndingLineThickness':0.15, 'textEnclosureThickness': 0.2}}";
+      "'hairpinThickness':0.1,'thickBarlineThickness':0.5,'tupletBracketThickness':0.1,'subBracketThickness':0.1,"
```

Comment:
Why are these values changed?

Reply:
The values should reflect default values of the corresponding options. Let's take an example. `thinBarlineThickness` is used to set the option `barLineWidth`. This happens in `Options::Sync`. The default value of `barLineWidth` is 0.3. There is always a conversion factor of 2 to MEI units involved. So `thinBarlineThickness` obtains the value 0.15. This scheme works fine for almost all entries in the list. It fails for `subBracketThickness`. This is used to set the option `subBracketThickness` which has default 0.2. If the old list value of 0.5 was used, then the option would be set to 1.0 (conversion factor). So I changed it to 0.1 in the list such that it works out. Similar for `repeatEndingLineThickness` and `textEnclosureThickness`.